Jenkinsfile

```diff
@@ -1140,11 +1140,12 @@ pipeline {
                                   rm -rf "Functional/"
                                   mkdir "Functional/"
                                   # compress those potentially huge DAOS logs
-                                  if daos_logs=$(ls install/lib/daos/TESTING/ftest/avocado/job-results/*/daos_logs/*); then
+                                  if daos_logs=$(find install/lib/daos/TESTING/ftest/avocado/job-results/*/daos_logs/* -maxdepth 0 -type f -size 2M); then
                                       lbzip2 $daos_logs
                                   fi
                                   arts="$arts$(ls *daos{,_agent}.log* 2>/dev/null)" && arts="$arts"$'\n'
                                   arts="$arts$(ls -d install/lib/daos/TESTING/ftest/avocado/job-results/* 2>/dev/null)" && arts="$arts"$'\n'
+                                  arts="$arts$(ls install/lib/daos/TESTING/ftest/*.stacktrace 2>/dev/null || true)"
```

Comment:
Why is this needed?  Stacktrace collection is done in `launch.py` so that stacktraces are filed with the test they occurred in.

Reply:
Some sections had this already, some didn't so I was making it consistent, I don't really know why it was different, do you have any ideas?

Reply:
Yeah, looks like #1886 probably was started before the multi-hardware-stage testing and didn't catch up when it merged.

So these should all be removed, not missing ones added.  Good catch noticing though.

And yes, it's not at all lost on us that this `Jenkinsfile` could use a lot of DRYing up.  John's got an experiment with matrix in `Jenkinsfile` that could achieve that.  Not sure I'm a fan of how it ends up running through.